src/libbz2.jl

```diff
@@ -58,55 +58,109 @@ function compress_init!(stream::BZStream,
                         blocksize100k::Integer,
                         verbosity::Integer,
                         workfactor::Integer)
+  if is_windows() && Sys.WORD_SIZE==32
```

Comment:
I'd rather suggest defining a constant like `const WIN32 = is_windows() && Sys.WORD_SIZE == 32` and use it anywhere needed.